app/Http/Controllers/PostController.php

```diff
@@ -82,6 +83,22 @@ public function update(PostRequest $request)
         return redirect()->route('posts.index')->with('success', 'Post updated successfully.');
     }
 
+    public function uploadFile(FileRequest $request)
+    {
+        $uploadMessage = $this->postService->uploadFile($request);
+
+        if (array_key_exists("error", $uploadMessage->original)) {
```

Comment:
`if (array_key_exists("error", $uploadMessage->original)) {`

to

`if (isset($uploadMessage['error'])) {`